experimental/regression_suite/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
+    "plat_host_cpu: mark tests as running on the host CPU",
+    "plat_rdna3_vulkan: mark tests as running on AMD RDNA3 Vulkan device",
```

Comment:
"plat" stands for "platform" here? That took me a little time to grok... are the 4 character savings worth it?